src/openrct2-ui/windows/TrackDesignPlace.cpp

```diff
@@ -503,38 +536,10 @@ class TrackDesignPlaceWindow final : public Window
             }
         }
 
-        // Draw entrance and exit preview.
-        for (const auto& entrance : td6->entrance_elements)
-        {
-            auto rotatedAndOffsetEntrance = origin + entrance.Location.ToCoordsXY().Rotate(rotation);
-
-            if (pass == 0)
-            {
-                min.x = std::min(min.x, rotatedAndOffsetEntrance.x);
-                max.x = std::max(max.x, rotatedAndOffsetEntrance.x);
-                min.y = std::min(min.y, rotatedAndOffsetEntrance.y);
-                max.y = std::max(max.y, rotatedAndOffsetEntrance.y);
-            }
-            else
-            {
-                auto pixelPosition = DrawMiniPreviewGetPixelPosition(rotatedAndOffsetEntrance);
-                if (DrawMiniPreviewIsPixelInBounds(pixelPosition))
-                {
-                    uint8_t* pixel = DrawMiniPreviewGetPixelPtr(pixelPosition);
-                    uint8_t colour = entrance.IsExit ? _PaletteIndexColourExit : _PaletteIndexColourEntrance;
-                    for (int32_t i = 0; i < 4; i++)
-                    {
-                        pixel[338 + i] = colour; // x + 2, y + 2
-                        pixel[168 + i] = colour; //        y + 1
-                        pixel[2 + i] = colour;   // x + 2
-                        pixel[172 + i] = colour; // x + 4, y + 1
-                    }
-                }
-            }
-        }
+        DrawMiniPreviewEntrances(*td6, pass, origin, min, max, rotation);
     }
 
-    void DrawMiniPreviewMaze(TrackDesign* td6, int32_t pass, const CoordsXY& origin, CoordsXY min, CoordsXY max)
+    void DrawMiniPreviewMaze(TrackDesign* td6, int32_t pass, const CoordsXY& origin, CoordsXY& min, CoordsXY& max)
```

Comment:
Why changing min and max to references? The caller doesn't seem to use the updated values.

Reply:
I mostly did it for consistency with `DrawMiniPreviewRide` and `DrawMiniPreviewEntrances`.